webapp/packages/core-blocks/src/FormControls/Checkboxes/CheckboxMarkup.css

```diff
@@ -0,0 +1,87 @@
+.checkbox-markup-dimensions_default {
```

Comment:
If we want to be aligned with UI-kit BEM notation, it's better to use -- (double dash) for modifiers. https://getbem.com/naming/

Reply:
By the way, why do we need that class at all? can we apply that on `.checkbox-markup` if it's default dimensions?